src/provider/cloud-element-templates/components/ElementTemplatesGroup.js

```diff
@@ -0,0 +1,297 @@
+    return <UnknownTemplate element={ element } />;
+  } else if (templateState.type === 'OUTDATED_TEMPLATE') {
+    return <OutdatedTemplate element={ element } templateState={ templateState } />;
+  }
+}
+
+function SelectEntryTemplate({ element }) {
+  const translate = useService('translate');
+  const eventBus = useService('eventBus');
+
+  const selectTemplate = () => eventBus.fire('elementTemplates.select', { element });
+
+  return (
+    <HeaderButton
+      title="SelectEntry a template"
+      class="bio-properties-panel-select-template-button"
+      onClick={ selectTemplate }
+    >
+      <CreateIcon />
+      <span>{ translate('Select') }</span>
+    </HeaderButton>
+  );
+}
+
+function AppliedTemplate({ element }) {
+  const translate = useService('translate'),
+        injector = useService('injector');
+
+  const menuItems = [
+    { entry: translate('Unlink'), action: () => unlinkTemplate(element, injector) },
+    { entry: <RemoveTemplate />, action: () => removeTemplate(element, injector) }
+  ];
+
+  return (
+    <DropdownButton menuItems={ menuItems } class="bio-properties-panel-applied-template-button">
+      <HeaderButton>
+        <span>{ translate('Applied') }</span>
+        <ArrowIcon class="bio-properties-panel-arrow-down" />
+      </HeaderButton>
+    </DropdownButton>
+  );
+}
+
+function RemoveTemplate() {
+  const translate = useService('translate');
+
+  return <span class="bio-properties-panel-remove-template">{ translate('Remove') }</span>;
+}
+
+function UnknownTemplate({ element }) {
+  const translate = useService('translate'),
+        injector = useService('injector');
+
+  const menuItems = [
+    { entry: translate('Unlink'), action: () => unlinkTemplate(element, injector) },
+    { entry: <NotFoundText /> },
+    { separator: true },
+    { entry: <RemoveTemplate />, action: () => removeTemplate(element, injector) }
+  ];
+
+  return (
+    <DropdownButton menuItems={ menuItems } class="bio-properties-panel-template-not-found">
+      <HeaderButton>
+        <span>{ translate('Not found') }</span>
+        <ArrowIcon class="bio-properties-panel-arrow-down" />
+      </HeaderButton>
+    </DropdownButton>
+  );
+}
+
+function NotFoundText() {
+  const translate = useService('translate');
+
+  return (
+    <div class="bio-properties-panel-template-not-found-text">
+      { translate(
+        'The template applied was not found. Therefore, its properties cannot be shown. Unlink to access the data.'
+      ) }
+    </div>
+  );
+}
+
+/**
+ *
+ * @param {object} props
+ * @param {object} element
+ * @param {UnknownTemplate} templateState
+ */
+function OutdatedTemplate({ element, templateState }) {
+  const { newerTemplate } = templateState;
+
+  const translate = useService('translate'),
+        injector = useService('injector');
+
+  const menuItems = [
+    { entry: translate('Update'), action: () => updateTemplate(element, newerTemplate, injector) },
```

Comment:
Same as before, consider same order as for platform. c.f. https://github.com/bpmn-io/bpmn-js-properties-panel/pull/545